cblite/EnrichCommand.cc

```diff
@@ -24,17 +24,20 @@
 #include "StringUtil.hh"
 #include "fleece/Mutable.hh"
 #include "Response.hh"
+#include "OpenAI.hh"
+#include "Gemini.hh"
 
 using namespace std;
 using namespace fleece;
 using namespace litecore;
 
 void EnrichCommand::usage() {
-    writeUsageCommand("enrich", false, "PROP");
+    writeUsageCommand("enrich", false, "PROPERTY DESTINATION");
     cerr <<
     "Enriches given JSON with embeddings of selected field\n"
     "    --offset N : Skip first N docs\n"
     "    --limit N : Stop after N docs\n"
+    "    --model NAME : AI model (NAME = 'openai', 'gemini', 'bedrock') (required)\n"
```

Comment:
These are not the models, these are the service providers.  The models are the things you pass to them like "model/amazon.titan-embed-text-v2:0", etc.  You'll need to have a system to work out which service provider to use based on the model name.